store/src/java/com/zimbra/cs/filter/SieveToSoap.java

```diff
@@ -463,6 +506,33 @@ protected void visitFlaggedTest(Node node, VisitPhase phase, RuleProperties prop
     }
 
     @Override
+    protected void visitAllOfTest(Node ruleNode, VisitPhase phase, RuleProperties props) {
+        visitLogicalTest(ruleNode, phase, props);
+    }
+
+    @Override
+    protected void visitAnyOfTest(Node ruleNode, VisitPhase phase, RuleProperties props) {
+        visitLogicalTest(ruleNode, phase, props);
+    }
+
+    private void visitLogicalTest(Node ruleNode, VisitPhase phase, RuleProperties props) {
```

Comment:
I intentionally defined this helper method as similar interface as other visit*Test() methods for the visibility purpose.